src/components/ProductCard/ProductCard.scss

```diff
@@ -0,0 +1,45 @@
+@import '/src/style/variables.scss';
+
+.item_card {
```

Comment:
you might wanna fix the class names too? - did you mean to add this stuff to this PR?